schemas/context/beacon-interaction-details.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/beacon-interaction-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Beacon",
+  "type": "object",
+  "description":
+    "Beacon, a wireless device that communicates identity information to mobile applications as mobile devices come within range.",
+  "definitions": {
+    "beacon-interaction-detail": {
+      "properties": {
+        "xdm:proximityUUID": {
+          "title": "Proximity UUID",
+          "type": "string",
+          "description": "The UUID configured into a beacon (or multiple beacons if multiple operate in an area) to be transmitted to mobile devices in range to identify an organizations beacons."
+        },
+        "xdm:beaconMajor": {
```

Comment:
can we give this another name, since major/minor usually mean version?

Reply:
In this case we are using the industry standard naming, to give it another name would put us out of the norm. 
https://en.wikipedia.org/wiki/IBeacon
```
Bluetooth low energy devices can operate in an advertisement mode to notify nearby devices of their presence.[26] At the most simple form, an iBeacon is a Bluetooth low energy device emitting advertisement following a strict format, that being an Apple defined iBeacon prefix, followed by a variable UUID, and a major, minor pair.[27] An example iBeacon advertisement frame could look like:

fb0b57a2-8228-44 cd-913a-94a122ba1206 Major 1 Minor 2
```